packages-exp/auth-exp/src/platform_cordova/popup_redirect/events.ts

```diff
@@ -30,6 +31,41 @@ import { browserLocalPersistence } from '../../platform_browser/persistence/loca
 
 const SESSION_ID_LENGTH = 20;
 
+/** Custom AuthEventManager that adds passive listeners to events */
+export class CordovaAuthEventManager extends AuthEventManager {
+  private readonly passiveListeners = new Set<(e: AuthEvent) => void>();
```

Comment:
This is mostly moved from popup_redirect.ts *except for the new `initPromise` field and the `initialized()` function*